docs/kubernetes/ingress-controller.md

```diff
@@ -281,3 +281,119 @@ Example ingress:
+
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name:  aws-ingress-controller
+      namespace: kube-system
+
+the required permissions are defined within a `ClusterRole` resource.
+
+_Note: It's important to use a `ClusterRole` instead of normal `Role` because otherwise Skipper could only access resources in the namespace the `Role` was created!_
+
+ClusterRole:
+
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRole
+    metadata:
+      name: aws-ingress-controller
+    rules:
+    - apiGroups: ["extensions"]
+      resources: ["ingresses", ]
+      verbs: ["get", "list"]
+    - apiGroups: ["extensions"]
+      resources: ["ingresses/status", ]
+      verbs: ["get", "list", "patch"]
+    - apiGroups: [""]
+      resources: ["namespaces", "services", "endpoints"]
+      verbs: ["get", "list"]
+
+This `ClusterRole` defines access to `get` and `list` all created ingresses, namespaces, services and endpoints.
+Furthermore it enables Skipper to `patch` the state of an ingress (like updating the endpoint of an ingress).
+
+To assign the defined `ClusterRole` to the previously created `ServiceAccount`
+a `ClusterRoleBinding` has to be created:
+
+ClusterRoleBinding:
+
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: aws-ingress-controller
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: aws-ingress-controller
+    subjects:
+    - kind: ServiceAccount
+      name: aws-ingress-controller
+      namespace: kube-system
+
+Last but not least the `ServiceAccount` has to be assigned to the Skipper daemonset.
+
+daemonset:
+
+    apiVersion: extensions/v1beta1
+    kind: DaemonSet
+    metadata:
+      name: skipper-ingress
+      namespace: kube-system
+      labels:
+        component: ingress
+    spec:
+      selector:
+        matchLabels:
+          component: ingress
+      updateStrategy:
+        type: RollingUpdate
+      template:
+        metadata:
+          name: skipper-ingress
+          labels:
+            component: ingress
+            application: skipper
+        spec:
+          hostNetwork: true
+          serviceAccountName: aws-ingress-controller
+          containers:
+          - name: skipper-ingress
+            image: registry.opensource.zalan.do/pathfinder/skipper:latest
+            ports:
+            - name: ingress-port
+              containerPort: 9999
+              hostPort: 9999
+            - name: metrics-port
+              containerPort: 9911
+            args:
+              - "skipper"
+              - "-kubernetes"
+              - "-kubernetes-in-cluster"
+              - "-address=:9999"
+              - "-proxy-preserve-host"
+              - "-serve-host-metrics"
+              - "-enable-ratelimits"
+              - "-experimental-upgrade"
+              - "-metrics-exp-decay-sample"
+            resources:
+              limits:
```

Comment:
I had a discussion with @aermakov-zalando on setting only limits or only requests. I think we agreed on having only requests, because it will remove the CFS settings at cgroup level, such that no throttling will happen.
Do I remember this correctly @aermakov-zalando or did we agreed on having only limits?

Reply:
For the ingress controller I think it's fine to set both because latency doesn't really affect it.